services/hierarchy/grpcapi/grpcapi.proto

```diff
@@ -61,8 +61,8 @@ message Node {
   AssetNode assetNode = 8;
   MeasurementPoint measurementPoint = 9;
   InspectionPoint inspectionPoint = 10;
-  PrimitiveInt64 childOrderNumber = 12;
-
+  int64 childOrderNumber = 11 [deprecated=true];
```

Comment:
it is not gonna work for sync App. Hierarchy  Node has slotnumber as 0 so i can not avoid it